test/api/v3/integration/tasks/POST-tasks_user.test.js

```diff
@@ -509,6 +509,7 @@ describe('POST /tasks/user', () => {
       expect(task.daysOfMonth).to.eql([15]);
       expect(task.weeksOfMonth).to.eql([3]);
       expect(new Date(task.startDate)).to.eql(now);
+      expect(task.isDue).to.be.true;
```

Comment:
Actually, what do these `isDue` changes have to do with PayPal IPN? Was this supposed to be part of a different branch?